### DISCUSSION TITLE
Doesn't Work - Showing ChatGPT is at capacity right now.

### DISCUSSION THREAD

**Discussion by yashpatel458** [lencx/ChatGPT#406] [Q&A]
ChatGPT is working fine in all my browsers but showing at capacity in the app. Please help.

<img width="946" alt="Screenshot 2023-02-16 at 5 21 10 PM" src="https://user-images.githubusercontent.com/68161473/219357595-c0c380ef-3e38-4051-aa3e-78347ee6ebaa.png">

**Comment by Erasliz** [comment]
its the same for everybody
